{
	"id": "CVE-2021-45469",
	"description": "In __f2fs_setxattr in fs/f2fs/xattr.c in the Linux kernel through 5.15.11, there is an out-of-bounds memory access when an inode has an invalid last xattr entry.",
	"scope": "local",
	"packages": [
		{
			"name": "linux",
			"status": "resolved",
			"urgency": "not yet assigned",
			"fixed_version": "4.19.232-1",
			"repository": [
				{
					"name": "buster",
					"version": "4.19.249-2"
				},
				{
					"name": "buster-security",
					"version": "4.19.304-1"
				}
			]
		}
	]
}
